hoki/data_compilers.py

```diff
@@ -1,19 +1,23 @@
 """
+Author: Max Briel & Heloise Stevance
+
 Objects and pipelines that compile BPASS data files into more convenient, more pythonic data types
 """
 
-from hoki.constants import DEFAULT_BPASS_VERSION, MODELS_PATH, OUTPUTS_PATH, dummy_dicts
-from hoki.load import model_input, dummy_to_dataframe
+import numpy as np
+
+from hoki.constants import (DEFAULT_BPASS_VERSION, MODELS_PATH,
+                            OUTPUTS_PATH, dummy_dicts,
+                            BPASS_IMFS, BPASS_METALLICITIES)
+import hoki.load
 import pandas as pd
 import re
 from hoki.utils.progressbar import print_progress_bar
 from hoki.utils.exceptions import HokiFatalError, HokiUserWarning, HokiFormatError, HokiKeyError
 from hoki.utils.hoki_object import HokiObject
 
 
-bpass_input_z_list = ['zem5','zem4', 'z001', 'z002', 'z003', 'z004', 'z006',
-                       'z008', 'z014', 'z010', 'z020','z030','z040']
-
+bpass_input_z_list = ["z"+i for i in BPASS_METALLICITIES]
```

Comment:
is there a reason the non numerical BPASS metalicities can't have the z in their string directly?

Reply:
No specific reason. More that z is constant between all metallicities, so there's no specific reason to have it be part of the list. Here I noticed that `bpass_input_z_list` is compared to user input, and I did not want to change that/didn't know where the user input was described. Therefore, I kept it like this. But we can change `BPASS_METALLICITIES` to contain the `z`. Although I'm not sure it should be part of it.

Reply:
I'm just thinking that having two lists is redundant is all

Reply:
What do we want? With or without? 
With means I go search for when I'm using it. 
Without means changing the tutorial for the DataCompiler.

Reply:
Let's meet at 10.50 on the zoom channel for the group meeting and find the best solution, will be easier than with comments :)

Reply:
Renamed to BPASS_METALLICITIES and now included "z"